requests/utils.py

```diff
@@ -456,15 +456,19 @@ def _parse_content_type_header(header):
 
     tokens = header.split(';')
     content_type, params = tokens[0].strip(), tokens[1:]
-    params_dict = {} 
+    params_dict = {}
+    items_to_strip = "\"' "
 
     for param in params:
-        if param and not param.isspace():
-            param = param.strip()
+        param = param.strip()
+        if param:
             key, value = param, True
-            if '=' in param:
-                param_tokens = [x.strip('\'" ') for x in param.split('=', 1)]
-                key, value = param_tokens[0], param_tokens[1]
+            index_of_equals = param.find("=")
+            if index_of_equals != -1:
+                before_equals = slice(0, index_of_equals)
+                after_equals = slice(index_of_equals + 1, len(param))
```

Comment:
Can you remove the `slice`s here? Just use index slice notation, (`param[:index_of_equals]` and `param[index_of_equals:]`) much more readable IMO.